kopf/cli.py

```diff
@@ -69,11 +77,18 @@ def run(paths, modules, peering_name, priority, standalone, namespace):
 @click.option('-n', '--namespace', default=None)
 @click.option('-i', '--id', type=str, default=None)
 @click.option('--dev', 'priority', flag_value=666)
-@click.option('-P', '--peering', 'peering_name', type=str, default=None, envvar='KOPF_FREEZE_PEERING')
-@click.option('-p', '--priority', type=int, default=100)
+@click.option('-P', '--peering', 'peering_name', type=str, required=True, envvar='KOPF_FREEZE_PEERING')
```

Comment:
With no `--peering` set, this CLI command was broken: it would try patching a resource like `/apis/zalando.org/v1/namespaces/ns1/kopfpeerings/None`.

Anyway, this is not something used in scripts normally, but more like for the development purposes: manually and on-demand. If used at all.

This change is just the alignment on how this command was supposed to be used: explicit peering name is needed.